pyproject.toml

```
@@ -74,7 +74,7 @@ align = ["uta-align~=0.3"]
 
 [project.urls]
 Documentation = "https://hgvs.readthedocs.io/"
-Homepage = "https://github.com/biocommons/hgvs"
+Homepage = "https://github.com/biocommons/hgvs/"
 Issues = "https://github.com/biocommons/hgvs/issues"
```

Not sure if the slash was added for consistency... if so
```suggestion
Issues = "https://github.com/biocommons/hgvs/issues/"
```

---

Yes, I try to remember to add those. (On some systems, it saves an http redirect delay). Thanks!